### DISCUSSION TITLE
Can't Install in Win 11 PC

### DISCUSSION THREAD

**Discussion by 100DavidX** [royshil/obs-backgroundremoval#318] [Q&A] [answered]
Installed OBS yesterday, did not do the update to latest version as prompted. Installed on Win 11, with current updates, Malwarebytes AV. Immediately downloaded the x64 add-in. Used installer. Background removal did not appear in the filters section of OBS. Then downloaded the zip package and copy/pasted the folders into the OBS file under programs. Still nothing in the filters section of OBS. Attempted 2, 3 times more. Uninstalled, rebooted, attempted everything again, this time ran the recommended update immediately after the OBS, the installed the update - still no filter. 

Can you help? 

**Desktop (please complete the following information):**

 - OS: \[WIN 11]
 - Browser: \[Chromeobs]
 - Plugin Version: \[e.g. 0.5.1\]
 - OBS Version: \[29.1
![No Filter](https://github.com/royshil/obs-backgroundremoval/assets/135154987/eef25eb2-7631-4d3f-b1c2-d921eb63cad9)
![OBS Tool](https://github.com/royshil/obs-backgroundremoval/assets/135154987/44ebec8a-2d72-4f31-a25e-2f2f719350ab)
]

**Additional context**

**Comment by umireon** [comment, marked as answer]
First, you have to add a video source to your scene.

<img width="1082" alt="スクリーンショット 2023-06-01 5 16 54" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/cce74e0b-76b6-4727-a530-b86a0d0c4ec3">

<img width="1191" alt="スクリーンショット 2023-06-01 5 17 43" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/f455c93c-ac8d-4f80-af54-1dbd2dbe08df">

<img width="832" alt="スクリーンショット 2023-06-01 5 17 59" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/9e8a2551-9304-4654-83c4-2c770c610a9f">

Second, you have to open the filter window of the video source.

<img width="1081" alt="スクリーンショット 2023-06-01 5 18 28" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/51fe6061-ea27-4767-a065-8ede70d6bd61">

<img width="973" alt="スクリーンショット 2023-06-01 5 19 15" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/3b377d10-7e31-4982-9a0d-963c0e4b496d">

Then, you have to click the + button under Effect filters and you will see the Background removal filter.
<img width="864" alt="スクリーンショット 2023-06-01 5 19 42" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/adfc2c87-ffe2-4864-8cd7-ad01da94fda9">

This filter can work without any configurations but you can change the configuration as you like.
<img width="973" alt="スクリーンショット 2023-06-01 5 20 28" src="https://github.com/royshil/obs-backgroundremoval/assets/1067855/3153974b-af38-4a26-b8d7-8fdeba6ab555">

If you have another question, please post on [Discussions](https://github.com/royshil/obs-backgroundremoval/discussions).

Thanks

**Comment by 100DavidX** [comment]
Thank you umireon, but your version is different than mine. I do have the video source installed, but when I pull up filters list, its very different from yours and does not include the background removal feature. Please see the screenshot of my list of filter options above.

**Reply by JaydzDev** [reply]
I had a similar issue...

Check to see if you have 2 obs-studio folders... one in "Program Files" and another in "Program Files (x86)".

In my case a fresh OBS went to one.
While the plugin went to the other.

I hope this helps. 🤞

**Comment by umireon** [comment]
My screenshot is correct and your screenshot is weird.
Please uninstall OBS and install the latest OBS from https://obsproject.com/download and try to open the same window as my screenshots.

**Comment by eth001** [comment]
same exact issue here, latest version of OBS, latest version of background remover (yesterday's version), tried installer, unzipped, nothing. Win11 as well.

**Reply by umireon** [reply]
Please try the installer https://github.com/royshil/obs-backgroundremoval/releases/download/v1.0.3/obs-backgroundremoval-1.0.3-windows-x64-Installer.exe

**Reply by umireon** [reply]
Note that this plugin will appear in Effect filters of video sources and the name is in your language.

**Reply by eth001** [reply]
like I stated, I tried the installer first (that particular link), nothing showed under filters

**Reply by umireon** [reply]
Can you create an issue with a filled template?
https://github.com/royshil/obs-backgroundremoval/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=
We need logs and screenshots to help you.
